### DISCUSSION TITLE
Time Domain Analysis

### DISCUSSION THREAD

**Discussion by hennryns** [jankae/LibreVNA#267] [General]
Hi @jankae , would LibreVNA be capable to do Time Domain Analysis?

**Comment by jankae** [comment]
Yes, and you can also test this without the VNA. Time domain transformations are available as a math operation and they can be used on any trace (either coming from live VNA data or imported from Touchstone/CSV files)

**Comment by hennryns** [comment]
Hi, this is good, I think that you should also advertise this capability for Time Domain Analysis, as follow:

![d36ac5d4-1fd5-4a8f-885d-7517c9a6e99b](https://github.com/user-attachments/assets/0210311a-241f-483a-9c6b-325519fc7199)

**Reply by jankae** [reply]
Yes, the feature is quite nice :)
But I have nothing to do with the advertising at all and to be honest I would probably not advertise the LibreVNA as in the picture you linked. Yes, it contains a spectrum analyzer and signal generator function. But it still primarily is a VNA. These other functions somewhat work and I added them to the software because I am a fan of hardware that is utilized as much as possible. They can be very helpful if you have no dedicated spectrum analyzer but they certainly will not reach the specifications of separate devices.

All functionalities of the LibreVNA are described in the user manual and I can not influence which features are used by sellers to advertise with.

**Reply by hennryns** [reply]
Hi, is the functionality about Time Domain Analysis also described in the user manual?

**Reply by jankae** [reply]
Of course, see section `3.4.2.2 TDR` for example. But you need to know the basics already. When I wrote the user manual, I assumed that the user already knows **what** he wants to do and I am just explaining **how** to do it with the LibreVNA.
